pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -555,7 +555,8 @@ def set_distributed_mode(self, distributed_backend: Optional[str] = None):
 
         rank_zero_info(f'GPU available: {torch.cuda.is_available()}, used: {self._device_type == DeviceType.GPU}')
         num_cores = self.tpu_cores if self.tpu_cores is not None else 0
-        rank_zero_info(f'TPU available: {_TPU_AVAILABLE}, using: {num_cores} TPU cores')
+        tpu_available = XLADeviceUtils.tpu_device_exists()
```

Comment:
We can use _TPU_AVAILABLE there.

Reply:
`_TPU_AVAILABLE` is replaced by `XLADeviceUtils.tpu_device_exists()`